pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>aws-secrets-manager-secret-source</artifactId>
-    <version>${revision}${changelist}</version>
+    <version>${revision}.${changelist}</version>
```

Comment:
changed your mind?

Reply:
Yep, count me under the category of "leery of committing up front to having major version numbers be in the triple digits, with no option of going back" :P